bundles/org.openhab.binding.tibber/src/main/java/org/openhab/binding/tibber/internal/handler/TibberHandler.java

```diff
@@ -101,9 +104,12 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
     public void getTibberParameters() {
         String response = "";
+        versionString = FrameworkUtil.getBundle(this.getClass()).getVersion().toString();
+        logger.debug("Binding version: {}", versionString);
         try {
             httpHeader.put("cache-control", "no-cache");
             httpHeader.put("content-type", JSON_CONTENT_TYPE);
+            httpHeader.put("user-agent", "openHAB/Tibber " + versionString + " com.tibber/1.8.3");
```

Comment:
Can version number "1.8.3" be extracted to a constant? What is the origin of this - is it API version?

Reply:
Yes, it is the API version. No known way to retrieve the API version dynamically, whereas the only reference is the Tibber Driver version (`com.tibber/1.8.3`) referred to in the API breaking changes documentation. So, my guess is that the Tibber Driver version will remain such until a new/potential breaking change will happen.

Reply:
OK. Still I would extract it as constant to be reused both places so they will not grow different over time.

Reply:
Moved to binding constants.